package.json

```diff
@@ -16,7 +16,7 @@
     "test:debug": "node --inspect-brk node_modules/.bin/jest --runInBand --no-watchman --config ./jest.config.cjs"
   },
   "peerDependencies": {
-    "graphql": "^0.8.0 || ^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0 || ^15.0.0 || ^16.0.0"
```

Comment:
do we actually need to drop other graphql versions to make the other changes in this PR?

Reply:
https://github.com/graphql/graphql-js/releases/tag/v16.0.0 sadly we can't do this easily.

Reply:
we can support 15 it seems, so 15 and 16 would be the graphql dependencies.